core/src/main/java/io/grpc/ManagedChannel.java

```diff
@@ -104,4 +104,22 @@ public ConnectivityState getState(boolean requestConnection) {
   public void notifyWhenStateChanged(ConnectivityState source, Runnable callback) {
     throw new UnsupportedOperationException("Not implemented");
   }
+
+  /**
+   * Notify the channel that a network connection is now available. This will attempt to invoke
+   * {@link NameResolver#refresh} and short-circuit the backoff timer for subchannel reconnection
+   * attempts.
+   *
+   * <p>This is primarily intended for Android users, where the network may experience frequent
+   * temporary drops. Rather than waiting for gRPC's name resolution and reconnect timers to elapse
+   * before reconnecting, the app may use this method as a mechanism to notify gRPC that a
+   * reconnection attempt may occur immediately.
+   *
+   * @throws UnsupportedOperationException if not supported by implementation
+   * @since 1.8.0
+   */
+  @ExperimentalApi
+  public void networkAvailable() {
```

Comment:
This method should be named in an imperative tone like the other methods. Maybe "reconnectNow"?

Reply:
Done